target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -141,6 +141,11 @@ tplink,archer-d50-v1)
 	ucidef_set_led_netdev "wan_data" "WAN Data" "tp-link:white:internet" "eth1.2" "tx rx"
 	ucidef_set_led_netdev "wan_link" "WAN Link" "tp-link:white:wan" "eth1.2" "link"
 	;;
+tplink,archer-d7-v1|\
+tplink,archer-d7b-v1)
+	ucidef_set_led_switch "lan" "LAN" "tp-link:white:lan" "switch0" "0x3c"
+	ucidef_set_led_wlan "wlan" "WLAN" "tp-link:white:wlan" "phy0tpt" "phy1tpt"
```

Comment:
Use `linux,default-trigger = "phy0tpt";` and `linux,default-trigger = "phy1tpt";` in your LED nodes instead. Configuration of LEDs in DTS is preferred. Example:
https://github.com/openwrt/openwrt/blob/e65f935de9e67ea07a251f114455f12e9555c5bf/target/linux/ath79/dts/ar9344_comfast_cf-e120a-v3.dts#L36-L40